packages/apple-pay-integration/e2e/applepay.spec.ts

```diff
@@ -63,7 +63,7 @@ test.describe('ApplePay', () => {
         await assertions.shouldSeeOrderConfirmation();
     });
 
-    test('Customer should be able to pay using ApplePay through the customer step in checkout', async ({
+    test.skip('Customer should be able to pay using ApplePay through the customer step in checkout', async ({
```

Comment:
I’ve marked this test as skipped because the test uses an outdated mock and no longer mimics the actual behaviour of ApplePay in Chrome. See https://bigcommerce.slack.com/archives/C09DAQUBL4Q/p1756446205795859?thread_ts=1756398135.742079&channel=C09DAQUBL4Q&message_ts=1756446205.795859. We need to either update the test and/or the mock to reflect the latest update to ApplePay.